src/demo/html/cdn.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>TinyMCE WebComponent Simple CDN Demo Page</title>
+    <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+    <script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@1/dist/tinymce-webcomponent.js"></script>
+  </head>
+
+  <body>
+  <h1>TinyMCE Web Component Simple CDN Demo Page</h1>
```

Comment:
Not sure if you're worried, but this doesn't match the title above (I believe the title is a typo).

Reply:
The same typo was on all the demo pages!